libraries/AP_Scripting/docs/docs.lua

```diff
@@ -340,6 +340,307 @@ function efi:get_state() end
 ---@return AP_EFI_Backend_ud
 function efi:get_backend(instance) end
 
+-- desc
+---@class airspeed
+airspeed = {}
+
+-- pass in external airspeed data
+---@param param1 airspeed_data_message_t_ud
+function airspeed:handle_external(param1) end
+
+
+-- desc
+---@class ExternalAHRS
+ExternalAHRS = {}
+
+-- desc
+---@param param1 EAHRS_State_ud
+---@param param2 nav_filter_status_flags_t_ud
+---@return boolean
+function ExternalAHRS:handle_scripting(param1, param2) end
+
+-- desc
+---@class mag_data_message_t_ud
+local mag_data_message_t_ud = {}
+
+---@return mag_data_message_t_ud
+function mag_data_message_t() end
+
+-- set field
+---@param value Vector3f_ud
+function mag_data_message_t_ud:field(value) end
```

Comment:
The generator gives you a default description all of the user data fields "set field" or "get field". So I think have a field called "field" is confusing. Maybe the object could be `compass_data_message` and the filed `mag_field`.